project/Build.scala

```diff
@@ -38,7 +38,20 @@ import org.scalajs.core.tools.linker.backend.OutputMode
 
 import sbtassembly.AssemblyPlugin.autoImport._
 
-object Build extends sbt.Build {
+/*
+ * In sbt 0.13 the Build trait would expose all vals to the shell, where you can use them in "set a := b"
+ * like expressions. This re-exposes them.
+ */
+object ExposedValues extends AutoPlugin {
+  override def requires = plugins.JvmPlugin
+  override def trigger = allRequirements
+  object autoImport {
+    val makeCompliant = Build.makeCompliant
+    val ES6NodeArgs   = Build.ES6NodeArgs
```

Comment:
I am less opinionated about the `build.sbt` in this case. I would lean towards following the coding style anyway, and not use vertical alignment there either.